scripts/travis.sh

```diff
@@ -64,7 +64,7 @@ fold_end
 # Test
 #
 fold_start "yarn-tests"
-yarn test
+xvfb-run --auto-servernum yarn test
```

Comment:
What is this?

Reply:
It is an replacement for the previous

```
  # Launch XVFB
  - "export DISPLAY=:99.0"
  - "sh -e /etc/init.d/xvfb start"
```

So instead of telling all processes that _there use the DISPLAY=:99.0_, you now prepend `xvfb-run` to the processes that need to use a display.